Description: define resolution to full-HD
as nowadays it's quite standard on modern computers
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2025-03-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/3dengfx.conf
+++ b/3dengfx.conf
@@ -4,8 +4,8 @@
 ; tool) so your best bet would be using that tool to configure the demo.
 ; If that does not work for some reason, modify this file with care...
 
-fullscreen = false
-resolution = 1024x768
+fullscreen = true
+resolution = 1440x1080
 bpp = dontcare
 zbuffer = dontcare
 stencil = dontcare
